presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -279,7 +279,7 @@ public void testParquetPageSource(int rowCount)
     {
         List<TestColumn> testColumns = getTestColumnsSupportedByParquet();
         testColumns = testColumns.stream()
-                .filter(column -> column.getObjectInspector().getCategory() == Category.PRIMITIVE)
+                .filter(column -> column.getObjectInspector().getCategory() == Category.PRIMITIVE || column.getName().equals("t_struct_bigint") || column.getName().equals("t_struct_null") || column.getName().equals("t_struct_non_nulls_after_nulls") || column.getName().equals("t_nested_struct_non_nulls_after_nulls"))
```

Comment:
this is a long predicate, how about creating a list with the new test column names and then do a contains check?